### ISSUE TITLE
Ｌｏｓｓ backward

### ISSUE THREAD

**Issue by dongda118** [Liuyixuan95/DORN#1]
Ｈｉ，
　Thanks for your code shared here!
    What I want to ask is : is there no need to rewrite the loss.backward() function?

Thanks,
